terraform/ecs/variables.tf

```diff
@@ -62,3 +62,8 @@ variable "ecs_extra_apps" {
 variable "kafka_version" {
   default = ""
 }
+
+// Source of the configurations in ECS: env_var, s3, http and https
+variable "configuration_source" {
```

Comment:
https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules
Should we start using validation rules to enforce that the value falls within the set defined in the comment?

Reply:
TIL. good idea.